contrib/depends/packages/boost.mk

```diff
@@ -4,6 +4,7 @@ $(package)_download_path=https://dl.bintray.com/boostorg/release/1.64.0/source/
 $(package)_file_name=$(package)_$($(package)_version).tar.bz2
 $(package)_sha256_hash=7bcc5caace97baa948931d712ea5f37038dbb1c5d89b43ad4def4ed7cb683332
 $(package)_dependencies=libiconv
+$(package)_patches=fix_aroptions.patch
```

Comment:
Just to note - this patch has now been merged into upstream boost. https://github.com/boostorg/build/pull/507